parser/scanner.l

```diff
@@ -692,6 +694,10 @@ year_month		{y}{e}{a}{r}_{m}{o}{n}{t}{h}
 			return curDate
 {current_date}		lval.item = string(l.val)
 			return currentDate
+{curtime}		lval.item = string(l.val)
```

Comment:
curtime is not MySQL keyword and can be used as identifier. So it should be added into NotKeywordToken rule in parser.y. Those tokens can be used as Identifier.
https://github.com/pingcap/tidb/blob/master/parser/parser.y#L1685

current_time is a reserved keyword. So it can not be used as identifier. It is ok now.